.happy/terraform/modules/sfn/main.tf

```diff
@@ -66,7 +68,111 @@ resource "aws_sfn_state_machine" "state_machine" {
               },
               {
                 "Name": "STEP_NAME",
-                "Value": "download-validate"
+                "Value": "download"
```

Comment:
I'm a little hazy on the details here, but can we confirm that we want `"Value"` here and that the above values are correctly either `"Value"` or `"Value.$"`?

Reply:
`"Value"` includes it as an env variable but doesn't add it to the Context object... is that right? Whereas `"Value.$"` _will_ add it to the Context object?

Reply:
"Value.$" to my understand will mean we are getting the value from a json object. The target would have to be `$.foo`